docs/dagster-university/pages/dagster-dbt/lesson-7/4-creating-the-manifest-with-github-actions.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Lesson 7: Creating the manifest with GitHub Actions"
+module: 'dbt_dagster'
+lesson: '7'
+---
+
+# Creating the manifest with GitHub Actions
+
+To recap, our deployment failed in the last section because Dagster couldn’t find a dbt manifest file, which it needs to turn dbt models into Dagster assets. This is because, during local development, we built this file by running `dbt parse`., as you learned in Lesson 3 and improved in Lesson 4. However, Dagster Cloud’s out-of-the-box `deploy.yml` GitHub Action isn’t aware that you’re also trying to deploy a dbt project with Dagster.
```

Comment:
```suggestion
To recap, our deployment failed in the last section because Dagster couldn’t find a dbt manifest file, which it needs to turn dbt models into Dagster assets. This is because we built this file by running `dbt parse` during local development. You ran this manually in Lesson 3 and improved the experience in Lesson 4. However, Dagster Cloud’s out-of-the-box `deploy.yml` GitHub Action isn’t aware that you’re also trying to deploy a dbt project with Dagster.
```